### ISSUE TITLE
What happened to this project? 

### ISSUE THREAD

**Issue by sebas77** [allisterb/jemalloc.NET#10]
Why was it abandoned suddenly?

**Comment by perlun** [comment]
Agree, activity on it seems to have just suddenly disappeared. :thinking: Which is a bit sad, for those of us who want/need an efficient way to allocate unmanaged memory for one reason or another.

Are there any other options out there nowadays? Even better, any options which are cross-platform an work on non-Windows platforms as well?
